crates/cairo-lang-casm/src/hints/mod.rs

```diff
@@ -333,6 +338,16 @@ impl Display for Hint {
                     "
                 )
             }
+            Hint::StartPrank { caller_address, target_contract_address , error_code} => {
+                writedoc!(
+                    f,
+                    "
+
+                    memory{error_code} = roll(caller_address={caller_address}, target_contract_address={target_contract_address}); 
```

Comment:
```suggestion
                    memory{error_code} = (caller_address={caller_address}, target_contract_address={target_contract_address}); 
```
This code probably should execute start_prank hint local

Reply:
right 🤦 [fix hint](https://github.com/software-mansion-labs/cairo/pull/14/commits/7da9d25a151f872ef21abae03342e7cc8b54e6d6)